contracts/solve/src/SolveInbox.sol

```diff
@@ -210,6 +226,9 @@ contract SolveInbox is OwnableRoles, ReentrancyGuard, Initializable, XAppBase, I
 
         req.updatedAt = uint40(block.timestamp);
```

Comment:
`updatedAt` is now redundant, we can just use last element of updateHistory.

Also, thoughts on just `history`?

Reply:
Agreed. I'll also use `history` instead.